openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java

```diff
@@ -103,7 +108,7 @@ void checkDuplicateWebhookName() throws InterruptedException {
     Events.click(webDriver, webhooks.allEntities());
     Events.click(webDriver, webhooks.clickToCloseDropdown());
     Events.click(webDriver, common.saveWebhook());
-    Thread.sleep(waitTime);
+    pause(waitTime);
```

Comment:
@kshinde2512 how is this different than what we had before. We are just moving thread.sleep to a method. Why aren't you using the await? @vivekratnavel

Reply:
@harshach This was just a temporary fix. Fluent wait and await was not working as expected so I had to put this with Fluent wait. But the replacement for this is also found and will merge the changes in PR shortly